iconsdk/signed_transaction.py

```diff
@@ -31,31 +31,27 @@ def __init__(self, transaction: Transaction, wallet: Wallet):
         :param transaction: A transaction object not having a signature field yet
         :param wallet: A wallet object
         """
-        self.__signed_transaction_dict = self.to_dict(transaction, wallet)
+        self.__signed_transaction_dict = self.convert_tx_to_jsonrpc_request(transaction, wallet)
         message_hash = sha3_256(serialize(self.__signed_transaction_dict)).digest()
         signature = wallet.sign(message_hash)
         self.__signed_transaction_dict["signature"] = b64encode(signature).decode()
 
     @property
-    def signed_transaction_dict(self):
+    def signed_transaction_as_dict(self):
```

Comment:
Does it need to change the name of signed_transaction_dict property?
When using signed_transaction_dict in a program, updating the icon-python-sdk causes the program to fail and not work.

Reply:
I missed that point. To make the meaning of the property of `signed_transaction_dict` clarify, I tried to change its name. And I've thought there are few users using it.

Reply:
It reverted to `signed_transaction_dict`.